integrations/cross_platform/android.rst

```diff
@@ -0,0 +1,200 @@
+.. _android:
+
+|android_logo| Android
+____________________________
+
+There are several ways to cross-compile packages for `Android <https://www.android.com/>`__ platform via conan.
+
+Using android_ndk_installer package (build require)
+=========================================================
+
+The easiest way so far is to use `android_ndk_installer <https://github.com/bincrafters/conan-android_ndk_installer>`_ conan package.
+
+Using the ``android_ndk_installer`` package as a build requirement will do the following steps:
```

Comment:
If it is not at conan-center (I haven't checked) show the command to add the remote. Otherwise comment that it is in the conan-center repository.

Reply:
it's in conan center, yes